lib/experimental/Widgets/Layout/Dashboard/index.tsx

```diff
@@ -42,7 +42,13 @@ const _Dashboard = forwardRef<HTMLDivElement, DashboardProps>(
       <div ref={ref} className="text-f1-foreground">
         <div ref={containerRef}>
           {columns === 1 ? (
-            <div className="flex flex-col gap-4">{children}</div>
+            <div className="flex flex-col gap-4">
+              {children?.map((child, i) => (
+                <span className="*:shadow" key={i}>
```

Comment:
why not adding the shadow selector to the class above, assuming that the only children allowed are widgets? 

`span` is not a good tag for the purpose because it is an inline element. It will be an invalid html if they will contain block elements, like _divs_, inside. If the above is not possible, let's at least use `div` instead of `span` here